src/components/aeAmountInput/aeAmountInput.vue

```diff
@@ -62,14 +62,24 @@
       }
     },
     data: () => ({
-      dropDownVisible: false
+      dropDownVisible: false,
+      unitsCount: null
     }),
     directives: { onClickAway },
     components: { AeInput, AeIcon },
     methods: {
       handleInput (value) {
         this.$emit('input', { ...this.value, ...value })
+      },
+      toggleDropDown () {
+        if (this.unitsCount > 1) {
+          this.dropDownVisible = !this.dropDownVisible
+        }
       }
+    },
+    mounted () {
+      this.value.symbol = this.units[0].symbol
+      this.unitsCount = this.units.length
```

Comment:
this could be done with a computed property, so if you have unit set dynamic it would also stay up to date.